### DISCUSSION TITLE
How to get the clock in system tray

### DISCUSSION THREAD

**Discussion by 4integration** [valinet/ExplorerPatcher#135] [Q&A] [answered]
Thanks for this feature, but for me the clock in the system tray disappeared .
How can I get that back?

**Comment by Gaurav-Original-ClassicShellTester** [comment, marked as answer]
Each big Windows update can reset the items in the system tray if the modern Taskbar loads. You can enable the missing applets by running: explorer.exe shell:::{05d7b0f4-2121-4eff-bf6b-ed3f69b894d9}\SystemIcons from the Run dialog. It's stated clearly in the README as well. This is not a problem with this software, it just that Windows decides to reset those.
